server/crawler/index.ts

```diff
@@ -255,5 +273,7 @@ export default async function getCralwerData(url: string) {
     // eslint-disable-next-line no-console
     console.error(error)
   }
+  if (page) await page.close();
+  if (browser) await browser.close();
```

Comment:
Can we declare `page` and `browser` only in the `if` block, use `const` and always close them?